### PR TITLE
OneScript 1.7.0

### PR DESCRIPTION
**Pull request by dmpas**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22556)

### CODE REVIEW

File: manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.ru-RU.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.7.0
+PackageLocale: ru-RU
+Publisher: OneScript
```
Comment (OfficialEsco) [inline manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.ru-RU.yaml:6]:
```suggestion
Publisher: 1Script Team (Open Source)
```

File: manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.7.0
+PackageLocale: en-US
+Publisher: OneScript
```
Comment (OfficialEsco) [inline manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.en-US.yaml:6]:
```suggestion
Publisher: 1Script Team (Open Source)
```

File: manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: OneScript.OneScript
+PackageVersion: 1.7.0
+DefaultLocale: ru-RU
```
Comment (OfficialEsco) [inline manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.yaml:5]:
```suggestion
DefaultLocale: en-US
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Please update your wingetcreate

File: manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.ru-RU.yaml (hunk 1 above)
Comment (dmpas) [reply manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.ru-RU.yaml:6]: should I also move manifest to `1/1Script Team (Open Source)/OneScript/` path ?

Comment (OfficialEsco) [reply manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.ru-RU.yaml:6]: Na there is no need for that :)

File: manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+PackageLocale: en-US
+Publisher: 1Script Team (Open Source)
+PackageName: OneScript
+License: MPL-2.0
+ShortDescription: OneScript engine
+Tags:
+- "Console"
+- "Command-Line"
+- "OneScript"
+- "oscript"
```
Comment (ImJoakim) [inline manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.en-US.yaml:14]: Best practice is to use all lower case. Referring to the 1.0.0 schema: [defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

```suggestion
- "console"
- "command-line"
- "onescript"
- "oscript"
```

File: manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.ru-RU.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+Publisher: 1Script Team (Open Source)
+PackageName: OneScript
+License: MPL-2.0
+ShortDescription: Движок OneScript
+Tags:
+- "Console"
+- "Command-Line"
+- "OneScript"
+- "oscript"
+- "ОСкрипт"
```
Comment (ImJoakim) [inline manifests/o/OneScript/OneScript/1.7.0/OneScript.OneScript.locale.ru-RU.yaml:15]: Best practice is to use all lower case. Referring to the 1.0.0 schema: [defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

```suggestion
- "console"
- "command-line"
- "onescript"
- "oscript"
- "оскрипт"
```

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**